src/main.js

```diff
@@ -0,0 +1,101 @@
+const play = document.getElementById('play')
+const pause = document.getElementById('pause')
+pause.disabled = true
+const clickMe = document.getElementById('clickMe')
+clickMe.disabled = true
+const timer = document.getElementsByClassName('controls__timer')[0]
+timer.textContent = '00:30'
+const contador = document.getElementsByClassName('progress')[0]
+const puntaje = document.getElementById('contador')
+contador.style.background = `linear-gradient(to right, var(--green) 0%, var(--green) 0%, #fff 00%, #fff 100%)`
+var tiempo = 30
+var clicks = 0
```

Comment:
Evitemos el uso de var ;) https://levelup.gitconnected.com/stop-using-var-to-declare-variables-in-javascript-6c0caec16f43
```suggestion
let tiempo = 30
let clicks = 0
```